kernel/memory/src/lib.rs

```diff
@@ -124,33 +124,34 @@ pub fn set_broadcast_tlb_shootdown_cb(func: fn(PageRange)) {
     BROADCAST_TLB_SHOOTDOWN_FUNC.call_once(|| func);
 }
 
-
+/// Information returned after initialising the memory subsystem.
+pub struct MemoryMappings {
```

Comment:
Thanks, this is way overdue lol. Things were out of hand.

Small request -- can we call it `InitialMemoryMappings` or something to imply bootstrap/init relevance?